### PR TITLE
M5Unified port

### PR DESCRIPTION
**Pull request by lvlhead**
I have been porting this library over to the M5Unified library instead of M5Stack in order to support a project I am working on.  At the moment it runs on an `M5Stack Core2 v1.1` with one issue in detecting the width/height inside of Layout.cpp.  This can be resolved by customizing the menu in the user code.

Please let me know if there are any issues with this approach.  Thanks!

```
Menu mainMenu("Main Menu");
customizeLayout(mainMenu.getLayout());
void customizeLayout(Layout& layout) {
    layout.SCREEN_WIDTH = M5.Display.width();
    layout.SCREEN_HEIGHT = M5.Display.height();
}
```

### CODE REVIEW

**Comment by lvlhead** [comment]
@chhorisberger Hope you've had a good week. Pinging you in this pull request :)

**Comment by chhorisberger** [comment]
@lvlhead Thanks for contributing! I'll check/test your proposed changes and will get back to you ASAP

**Review by chhorisberger — CHANGES_REQUESTED** [review]

**Review by chhorisberger — APPROVED** [review]

File: src/CallbackMenuItem.cpp [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 #include "Menu.h"
```
[the remark this answers is not in the document]
Comment (chhorisberger) [reply src/CallbackMenuItem.cpp:4]: Update: On a second though, I'll just replace those myself after merging your pull request ;)

**Comment by chhorisberger** [comment]
Also, I found a fix for the issue you mentioned about screen width and height.
The point is that after the switch to M5Unified, the width and height can only be determined at runtime.
The solution was to create an "init" method and get/set them there

**Comment by chhorisberger** [comment]
Release is done. Just a matter of waiting a few hours, until the ardunio library registry picks them

https://downloads.arduino.cc/libraries/logs/github.com/chhorisberger/M5StackMenuSystem/

**Comment by mdxs** [comment]
Excellent, thanks for working on this. Great idea to port it to M5Unified!
I hope to get around to testing this implementation later this week.

**Comment by chhorisberger** [comment]
Version 1.2.0 now available in Arduino IDE

**Comment by lvlhead** [comment]
@chhorisberger Thank you for an amazing library!

**Pull request merged**